### PR TITLE
[4.0] Remove "(TEMPORARY)" css rules in main cassiopea template.scss

### PR DESCRIPTION
**Pull request by niharikamahajan02**
Pull Request for Issue #35714 .

### Summary of Changes
Removed "temporary" css rules as mentioned in issue.

### CODE REVIEW

**Comment by richard67** [comment]
@niharikamahajan02 Could you give this pull request a more meaningful title? Maybe something like

[4.0] Remove "(TEMPORARY)" css rules in main cassiopea template.scss

And maybe provide some testing instructions, too.

Thanks in advance.

**Comment by niharikamahajan02** [comment]
Done with the required changes.

File: administrator/templates/atum/scss/blocks/_global.scss [hunk 1]
```diff
@@ -154,10 +154,6 @@ body:not(.contentpane) .main-card {
   background-color: var(--toolbar-bg);
 }
 
-// Chosen (TEMPORARY)
-.chzn-container-single {
-  width: auto !important;
-}
```
Comment (Quy) [inline administrator/templates/atum/scss/blocks/_global.scss]:
```suggestion
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on cf42afa1c39edd28c74c8f1392c570ca687d191b<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36596">issues.joomla.org/tracker/joomla-cms/36596</a>.</sub>

**Comment by Quy** [comment]
Please fix conflicts.

**Comment by niharikamahajan02** [comment]
> Please fix conflicts.

I am not able to know what the conflict is in these 2 files .
administrator/templates/atum/scss/blocks/_global.scss
templates/cassiopeia/scss/template.scss

May I know these conflicts please.

**Comment by richard67** [comment]
> I am not able to know what the conflict is in these 2 files . administrator/templates/atum/scss/blocks/_global.scss templates/cassiopeia/scss/template.scss
> 
> May I know these conflicts please.

@niharikamahajan02 In general you will not see any conflicts when just checking your files or when just checking the changes on GitHub.

Conflicts (which means merge conflicts, not political conflicts 😄 ) will appear when you have made changes in some file, and in the base branch of your branch other changes have been made in the same file at the same place.

Here we had to change the base branch of your PR from 4.0-dev to 4.1-dev because 4.0 has end of development, and 4.1 is the current version soon, so development (like bug fixes) continues on that version and so on the 4.1-dev branch, and new features will go into 4.2.

With version 4.1 the 2 files which have the conflicts have been moved from the template to the media folder because the templates have been changed in 4.1 to support child templates, and that's the conflict now:
- "administrator/templates/atum/scss/blocks/_global.scss" has been moved to "build/media_source/templates/administrator/atum/scss/blocks/_global.scss"
- "templates/cassiopeia/scss/template.scss" has been moved to "build/media_source/templates/site/cassiopeia/scss/template.scss"

(When the scss files are compiled the results will be not in "build/media_source" but in the same place below "media", but this is just for your understanding, it is not relevant for solving the conflict.)

If a conflict is easy, there is a button on GitHub so solve it there in an editor. But in this case the conflict seems to be too complicated for the GitHub web site, so the button is disabled.

So these conflicts have to be solved in your local git clone with help of the git command line (normally a CMD window on Windows and a Bash shell window on Linux).

The conflicts are then solved by merging the 4.1-dev branch of the upstream repository (this one here) into your branch for this PR. Git will then tell you that there are conflicts which you have to solve. In this case here you solve them by applying the changes which have made here in the files at the old place to the files at the new place and accept that the old files are deleted.

You can find some documentation on how to solve conflicts here: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-using-the-command-line

If you want to learn that - and I think you should because being familiar with come code management and version control system like Git is a key skill for people who develop software in companies, teams, as soon as more than 1 person works on a software - then try to solve it yourself,

If you run into problems and it does not work for you, either call here for help, or (maybe the easiest way if you messed this branch with trying to solve conflicts) you make a new PR for the 4.1-dev branch which makes the changes from here but at the files at their new locations.

**Comment by niharikamahajan02** [comment]
@richard67 sir thanks for telling . Are the commits that I made now correct?

**Comment by niharikamahajan02** [comment]
@richard67 sir this concept is new to me , please tell if the commit i did is correct , thanks.

**Comment by richard67** [comment]
@niharikamahajan02 I see you have created a new PR #36897 . Does that PR replace this one here? If so, why haven’t you closed this one here. Do you really think it helps us if people keep their duplicate PRs open so we have to identify them and close the duplicates?

**Comment by niharikamahajan02** [comment]
ok @richard67 sir , I've closed that PR , this is the final PR  .Thanks

**Comment by HLeithner** [comment]
This pull request has automatically rebased to 4.2-dev.

**Comment by brianteeman** [comment]
It is not possible any more to see why the drone build for this pr was failing but perhaps it is because the tests need to be updated to remove this test
https://github.com/joomla/joomla-cms/blob/04bf6459a7bd718d6727a4f0335306f21c3e841f/tests/Codeception/_support/Page/Acceptance/Administrator/AdminPage.php#L189

**Comment by bembelimen** [comment]
Thx

**Pull request merged**